packages/chai-a11y-axe/chai-a11y-axe-plugin.d.ts

```diff
@@ -2,11 +2,11 @@
 
 declare namespace Chai {
   interface Assertion {
-    accessible(options?: Object): Assertion;
+    accessible(options?: Object): Promise<Assertion>;
   }
 
   interface Assert {
-    isAccessible(fixture: any, options?: Object): Assertion;
-    isNotAccessible(fixture: any, options?: Object): Assertion;
+    isAccessible(fixture: any, options?: Object): Promise<Assertion>;
+    isNotAccessible(fixture: any, options?: Object): Promise<Assertion>;
```

Comment:
I wonder why I've never run into issues using `await expect(el).to.be.accessible();` in TS before. Do you think this is a general thing or something specific to your usage?

Reply:
I should mention that it's tsserver and it only gives a warning.  Regular compile from CLI with `tsc` doesn't have an issue

Reply:
Hrm. I wonder what’s different between the two. Does this change work fine with the CLI? Haven’t had a chance to confirm.